cmd/collectors/rest/plugins/netroute/netroute.go

```diff
@@ -69,11 +70,13 @@ func (n *NetRoute) Run(dataMap map[string]*matrix.Matrix) ([]*matrix.Matrix, err
 	for key, instance := range data.GetInstances() {
 		cluster := data.GetGlobalLabels().Get("cluster")
 		routeID := instance.GetLabel("uuid")
-		interfaceName := instance.GetLabel("interface_name")
-		interfaceAddress := instance.GetLabel("interface_address")
-		if interfaceName != "" && interfaceAddress != "" {
-			names := strings.Split(interfaceName, ",")
-			address := strings.Split(interfaceAddress, ",")
+		interfaces := instance.GetLabel("interfaces")
+
+		interfacesList := gjson.Result{Type: gjson.JSON, Raw: interfaces}
+		names := interfacesList.Get("name").Array()
+		address := interfacesList.Get("address").Array()
+
+		if names != nil && address != nil {
```

Comment:
The result of `Array()` can never be null, the `if len()` checks are sufficient

Reply:
Done